### PR TITLE
add lib for Mac OS

### PR DESCRIPTION
**Pull request by Fan1117**
Adding folder and subfolder paths is required when using MATLAB packages. Tested on MATLAB R2019a.

### CODE REVIEW

File: matlab-package/brainflow/brainflow_get_data.m [hunk 1]
```diff
@@ -1,4 +1,4 @@
-board_shim = BoardShim (BoardsIds.CYTON_BOARD, '/dev/ttyUSB0')
+board_shim = BoardShim (BoardsIds.CYTON_BOARD, '/dev/tty.Bluetooth-Incoming-Port')
```
Comment (Andrey1994) [inline matlab-package/brainflow/brainflow_get_data.m]: is it a port name for Mac?

Comment (Fan1117) [reply matlab-package/brainflow/brainflow_get_data.m]: Yes, I change path manually in  a matlab's gui. Yes, this is a port name for Mac. However, the Mac I used to test package can't recognize the driver. So I use a port name that can be recognized by that Mac.

Comment (Fan1117) [reply matlab-package/brainflow/brainflow_get_data.m]: Just matlab-package is OK for running. No need project file.

**Comment by Andrey1994** [comment]
thanks!

**Comment by Fan1117** [comment]
My pleasure :)

**Pull request merged**